### DISCUSSION TITLE
Why YCbCr color space shown as sRGB ?

### DISCUSSION THREAD

**Discussion by Nikita-schetko** [ImageMagick/ImageMagick#7220] [Help]
### ImageMagick version

7.1.1-29

### Operating system, version and so on

MacOs 14.1.1

### Description

Hello, ImageMagick Team!
I'm currently experiencing an issue where the output from the identify command does not reflect the expected colorspace after converting an image's colorspace with the convert command. Here are the details of my process:

`convert first_frame.jpg -colorspace YCbCr first_frame_yCbCr.jpg`
I used this command with the expectation that it would convert the colorspace of first_frame.jpg to YCbCr. However, when I inspect the converted   image using:
`identify -verbose first_frame_yCbCr.jpg`
The output indicates that the colorspace is still sRGB instead of YCbCr.
`Colorspace: sRGB`

 Context:
My original objective was to replicate the exact appearance of the first frame from a video encoded in h265/yuv420p. When extracting the frame with FFmpeg, I noticed the colors were slightly different and a bit darker compared to the original video playback. Hence, I attempted to use ImageMagick to adjust the colorspace, aiming to match the frame's appearance to that of the video.
Could you please provide some insights or guidance on why the colorspace conversion isn't reflected in the identify output? Am I missing something in my understanding or approach?
Thank you for your time and assistance!

```

frame=    1 fps=0.0 q=7.8 Lsize=N/A time=00:00:00.00 bitrate=N/A speed=   0x    
Image:
  Filename: first_frame_yCbCr.jpg
  Permissions: rw-r--r--
  Format: JPEG (Joint Photographic Experts Group JFIF format)
  Mime type: image/jpeg
  Class: DirectClass
  Geometry: 1920x1010+0+0
  Units: Undefined
  Colorspace: sRGB
  Type: TrueColor
  Base type: Undefined
  Endianness: Undefined
  Depth: 8-bit
  Channels: 3.0
  Channel depth:
    Red: 8-bit
    Green: 8-bit
    Blue: 8-bit
  Channel statistics:
    Pixels: 1939200
    Red:
      min: 0  (0)
      max: 255 (1)
      mean: 27.7807 (0.108944)
      median: 10 (0.0392157)
      standard deviation: 54.8413 (0.215064)
      kurtosis: 7.81677
      skewness: 2.96658
      entropy: 0.635464
    Green:
      min: 0  (0)
      max: 255 (1)
      mean: 28.0174 (0.109872)
      median: 10 (0.0392157)
      standard deviation: 55.4041 (0.217271)
      kurtosis: 7.77439
      skewness: 2.96143
      entropy: 0.634922
    Blue:
      min: 0  (0)
      max: 255 (1)
      mean: 26.9057 (0.105513)
      median: 8 (0.0313725)
      standard deviation: 55.2373 (0.216617)
      kurtosis: 7.86266
      skewness: 2.9831
      entropy: 0.626025
  Image statistics:
    Overall:
      min: 0  (0)
      max: 255 (1)
      mean: 27.5679 (0.10811)
      median: 9.33333 (0.0366013)
      standard deviation: 55.1609 (0.216317)
      kurtosis: 7.81794
      skewness: 2.97037
      entropy: 0.632137
  Rendering intent: Perceptual
  Gamma: 0.454545
  Chromaticity:
    red primary: (0.64,0.33,0.03)
    green primary: (0.3,0.6,0.1)
    blue primary: (0.15,0.06,0.79)
    white point: (0.3127,0.329,0.3583)
  Matte color: grey74
  Background color: white
  Border color: srgb(223,223,223)
  Transparent color: black
  Interlace: None
  Intensity: Undefined
  Compose: Over
  Page geometry: 1920x1010+0+0
  Dispose: Undefined
  Iterations: 0
  Compression: JPEG
  Quality: 72
  Orientation: Undefined
  Properties:
    comment: Lavc60.31.102
    date:create: 2024-04-04T12:34:12+00:00
    date:modify: 2024-04-04T12:34:12+00:00
    date:timestamp: 2024-04-04T12:34:12+00:00
    jpeg:colorspace: 2
    jpeg:sampling-factor: 2x2,1x1,1x1
    signature: c1a39fb0245aa24c4f78d540a48212d193abea79c8c3f6558557706d2d0d5ec9
  Artifacts:
    verbose: true
  Tainted: False
  Filesize: 62415B
  Number pixels: 1.9392M
  Pixel cache type: Memory
  Pixels per second: 133.22MP
  User time: 0.020u
  Elapsed time: 0:01.014
  Version: ImageMagick 7.1.1-29 Q16-HDRI x86_64 21991 https://imagemagick.org   

```

**Comment by snibgo** [comment]
When IM writes JPEG files, they are always encoded as YCbCr. IM silently encodes from sRGB to YCbCr. When IM reads JPEG, it silently converts from YCbCr to sRGB.

I suggest you experiment with a different file format such as TIFF.

**Comment by urban-warrior** [comment]
Try this command: 

```
magick -verbose -set colorspace YCbCR first_frame_yCbCr.jpg
```
